src/foam/u2/view/TreeView.js

```diff
@@ -197,8 +197,8 @@ foam.CLASS({
       var M   = this.ExpressionsSingleton.create();
       var of  = this.lookup(this.relationship.sourceModel);
       var dao = this.data$proxy.where(
-        M.NOT(M.HAS(of.getAxiomByName(this.relationship.inverseName))));
-
+        M.NOT(M.EQ(of.getAxiomByName(this.relationship.inverseName), null)));
+      
```

Comment:
Remove trailing whitespace.

Reply:
Just revert this line and I'll merge the rest of the code, since this is unrelated to the DAOController anyway.